create-topics.sh

```diff
@@ -27,15 +27,26 @@ if $start_timeout_exceeded; then
 fi
 
 # Expected format:
-#   name:partitions:replicas:cleanup.policy
-IFS=','; for topicToCreate in $KAFKA_CREATE_TOPICS; do
+#   name:partitions:replicas:cleanup.policy:<key>=<value>:<key>=<value>...
+IFS=';'; for topicToCreate in $KAFKA_CREATE_TOPICS; do
     echo "creating topics: $topicToCreate"
     IFS=':' read -r -a topicConfig <<< "$topicToCreate"
-    config=
-    if [ -n "${topicConfig[3]}" ]; then
-        config="--config=cleanup.policy=${topicConfig[3]}"
-    fi
-    JMX_PORT='' "$KAFKA_HOME/bin/kafka-topics.sh" --create --zookeeper "$KAFKA_ZOOKEEPER_CONNECT" --replication-factor "${topicConfig[2]}" --partitions "${topicConfig[1]}" --topic "${topicConfig[0]}" "$config" --if-not-exists &
+    config=""
+    for i in {3..100}
+    do
+        if [ -n "${topicConfig[i]}" ]; then
+            config="${config} --config=${topicConfig[i]}"
+        fi
+    done
+    COMMAND="JMX_PORT='' ${KAFKA_HOME}/bin/kafka-topics.sh
```

Comment:
I think if you move to the eval approach, then my earlier suggestion of 

```
topic:1:2:compact --config=compression.type=snappy
```

Works fine - negating the need for this to be a breaking change.

However, this should still have a test-case added to verify this behaviour